### DISCUSSION TITLE
BackTop 回到顶部如何嵌入组件里使用

### DISCUSSION THREAD

**Discussion by allen1696** [vueComponent/ant-design-vue#6865] [Q&A]
div的内容过长，想在组件里面的div嵌入BackTop回到div容器的顶部，target怎么写？试过很多方法，默认的window可以用，在组件里却用不了

**Comment by selicens** [comment]
![image](https://github.com/vueComponent/ant-design-vue/assets/69418751/3891d800-8a30-49af-8d7f-9b77d40b8b0f)
